src/index.html

```diff
@@ -1,13 +1,308 @@
+          <p class="page__section--comment">Explore exhibition gallery guides, videos, <br> articles, and more.</p>
+
+          <div class="gallery__place place">
+            <div class="place__photos place__photos--big-small">
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/1.png"
+                  alt="picture"
+                  class="place__photo"
+                >
+              </div>
+
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/2.png"
+                  alt="portret"
+                  class="place__photo"
+                >
+              </div>
+            </div>
+            <p class="place__description--first">A New Look at Old Masters</p>
+            <p class="place__description--second">December 12, 2020 – Through Spring 2023</p>
+
+            <div class="place__photos place__photos--equals">
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/3.png"
+                  alt="picture"
+                  class="place__photo"
+                >
+              </div>
+
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/4.png"
+                  alt="portret"
+                  class="place__photo"
+                >
+              </div>
+
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/5.png"
+                  alt="portret"
+                  class="place__photo"
+                >
+              </div>
+            </div>
+            <p class="place__description--first">Bodhisattvas of Wisdom, Compassion, and Power</p>
+            <p class="place__description--second">March 27, 2021 – October 30, 2022</p>
+
+            <div class="place__photos place__photos--small-big">
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/6.png"
+                  alt="picture"
+                  class="place__photo"
+                >
+              </div>
+
+              <div class="place__photo--container">
+                <img
+                  src="./images/photos/7.png"
+                  alt="portret"
+                  class="place__photo"
+                >
+              </div>
+            </div>
+
+            <p class="place__description--first">Inspiring Walt Disney: The Animation of French Decorative Arts</p>
+            <p class="place__description--second">December 10, 2021 – March 6, 2022</p>
+          </div>
+
+          <div class="place__button">
+            <div class="button">
+              <a href="#" class="button--link">View all exhibitions</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section testimonials" id="Testimonials">
```

Comment:
you also got horizontal scroll cause of these pictures
![image](https://user-images.githubusercontent.com/37161315/170643088-385f36cb-666b-45f1-9d43-f58c27fdd01e.png)